README.md

```diff
@@ -4,7 +4,26 @@
 [![Chat](https://img.shields.io/static/v1.svg?label=chat&message=mattermost&color=145dbf)](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind)
 
 # Codewind for VS Code
-Create and develop cloud-native, containerized web applications from VS Code.
+Codewind for VS Code provides tools to help build cloud-native, containerized applications from VS Code, regardless of which runtime or language you use. Rapidly create an application from a template project and launch, update, test, and debug in Docker containers on the desktop.
+
+## Why use Codewind?
+- Get started quickly with templates or samples.
+- Use the tools, runtimes, and languages that you're already familiar with.
+- Pull in existing applications and get them ready for the cloud.
+- Enjoy quick, iterative development and fast feedback.
```

Comment:
Our advanced inner loop ensures that we manage your application updates efficiently, so that your code changes are reflected in your containerized application almost instantaneously.